src/config/graphical-session/xorg.md

```diff
@@ -35,10 +35,10 @@ of device brands and their driver packages.
 | Intel  | Open Source | `xf86-video-intel`   |
 | NVIDIA | Open Source | `xf86-video-nouveau` |
 
-> Note: Fourth generation Intel users may want to use the default xorg driver,
-> rather than installing `xf86-video-intel` driver package. For more
-> information, see the [Arch wiki
-> page](https://wiki.archlinux.org/index.php/Intel_graphics#Installation).
+Note: Fourth generation Intel users may want to use the default xorg driver,
+rather than installing `xf86-video-intel` driver package. For more information,
+see the [Arch wiki
+page](https://wiki.archlinux.org/index.php/Intel_graphics#Installation).
```

Comment:
This note is confusing. What is the default xorg driver, xorg-minimal? how do you use the default driver if you installed the whole xorg metapackage? The linked archwiki is not a great resource in this case because it also doesn't explain what the issue with xf86-video-intel is and just points to decisions made by other distributions. We should also give a year reference for "fourth generation". Archwiki says everything since 2006, should this be phrased as "fourth generation and newer", or is the current stuff still "fourth generation"?

Reply:
@st3r4g: Any thoughts on this?

Reply:
This is exactly the same stuff already discussed for nouveau, but for Intel: Xorg can use the newer 'modesetting' driver (needs `mesa-dri`) or the legacy 'intel' driver (needs `xf86-video-intel`). Nowadays modesetting works well and it's the generally recommended option afaik, most distros default to that. But note that if you have `xf86-video-intel` installed, Xorg will pick that by default (you need a Xorg config file to pick modesetting).
So the user has to make a choice here, and the default on Void seems to still be 'intel' (since `xorg` ships the `xf86-*` drivers). I don't really feel entitled to dictate what should be the default, since I'm using Wayland now. Which one is better depends on the particular graphics card, I'd suggest to try both and stick with the one that works best for you (no tearing, etc...)

Reply:
I think we need a paragraph in the xorg section explaining that Xorg can run with two different kinds of graphics drivers: 'modesetting' vs 'DDX', and how to check what you're running.

Reply:
I think this part of the Xorg is nearly mergeable, so we should probably link to it.

Reply:
Merged, gotta rebase this one to remove it :D 

Thanks again @st3r4g

Reply:
good stuff :)